apostrophecms/cli
# Add flag to allow use of hosted MongoDB

## Summary

*Summarize the changes briefly, including which issue/ticket this resolves. If it closes an existing Github issue, include "Closes #[issue number]"*
This PR is in response to a request by a dev in Discord and developers at Brotherhood Mutual asking for better documentation for connecting to a non-local MongoDB server. Additionally, there has been some internal discussion of pointing people at the free tier of MongoDB Atlas to ease development setup. The issue is that our CLI will fail to create a user in this case since it is looking for the localhost server. Once this lands, the documentation will be updated. Closes PRO-5320

## What are the specific steps to test this change?

*For example:*
> 1. Run the website and log in as an admin
> 2. Open a piece manager modal and select several pieces
> 3. Click the "Archive" button on the top left of the manager and confirm that it should proceed
> 4. Check that all pieces have been archived properly
1. Create a MongoDB Atlas account and provision a database.
2. Create a new project using the `--uri` flag and the connection string from your account.
3. Start your project with the `APOS_MONGODB_URI` environment variable set to the Atlas connect stream and verify you can login with the credentials set during project creation.
## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:

- [1/2 ] Related documentation has been updated

If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
